instrumentation/common-api/src/main/java/io/opentelemetry/android/instrumentation/annotations/RumScreenName.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.annotations
+
+/**
+ * This annotation can be used to customize the `screen.name` attribute for an instrumented
+ * Fragment or Activity.
+ */
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS)
+annotation class RumScreenName(
+    /** Return the customized screen name.  */
+    val value: String,
+)
```

Comment:
Do we know if migrating this annotation from java to kotlin will break existing java users?